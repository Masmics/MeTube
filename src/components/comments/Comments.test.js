import React from 'react';
import { shallow } from 'enzyme';
import Comments from './Comments';

describe('comments list component', () => {
  it('renders a list of video comments', () => {
    const wrapper = shallow(<Comments comments={['Sweet video!', 'Loved it']} />);
    expect(wrapper).toMatchSnapshot();
  });
});
