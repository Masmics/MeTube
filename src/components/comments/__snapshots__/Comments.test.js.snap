// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`comments list component renders a list of video comments 1`] = `
<ul>
  <li
    key="0"
  >
    <Comment
      comment="Sweet video!"
    />
  </li>
  <li
    key="1"
  >
    <Comment
      comment="Loved it"
    />
  </li>
</ul>
`;
